clients/src/main/java/org/apache/kafka/common/requests/OffsetCommitRequest.java

```diff
@@ -154,8 +153,8 @@
     private final Map<TopicPartition, PartitionData> offsetData;
 
     public static final class PartitionData {
-        @Deprecated
-        public final long timestamp;                // for V1
+        // Note, only available for OffsetCommitRequest v1
+        public final long timestamp;
```

Comment:
Removed the `@Deprecated` annotation since both the Kafka broker and the Kafka client need to support old version of the RPC.